.github/workflows/checks.yml

```diff
@@ -13,30 +13,44 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
           cache: 'npm'
       - run: npm run install-ci
       - run: npm run lint
 
-  unit:
+  build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: [18, 19]
```

Comment:
Maybe have 16 in the list too? My system is still on 16 at least.

Reply:
I wanted to avoid testing too many combinations. Either we go with the two most common versions in use (probably 16 and 18), or test against LTS + current (18 and 19), so we get an early warning of future build breaking issues. Thinking about it, maybe 16 and 18 makes more sense here?